tests/Backend/SOX/PullTableTest.php

```diff
@@ -33,18 +31,6 @@ public function setUp(): void
         }
     }
 
-    private function getBranchesForCurrentTestCase(): array
```

Comment:
Tu som to po rebase tiez zmazal

Reply:
Ty vole to je dost divnost, ne? říkáš, že ve tests/StorageApiTestCase.php je property protected DevBranches $branches; ale ona tam není a setupuješ ju v setup metodě classy co to dědí :exploding_head: :gun:

Reply:
udělal bych teda `getBranchesForCurrentTestCase(Client): array` a tu tokens instanci si nasetupoval tam.

Reply:
Hm ok no tak by to slo nejakou traitu :)

Reply:
No je to z bláta do louže, ale asi jo :)

Reply:
Upravil som to.

Reply:
:D ale nezlepšils to :D furt tam máš this->branches to je pořád na prd, myslel sem aby ta property zmizela.